api/cas-server-core-api-events/src/main/java/org/apereo/cas/support/events/logout/CasRequestSingleLogoutEvent.java

```diff
@@ -0,0 +1,34 @@
+package org.apereo.cas.support.events.logout;
+
+import org.apereo.cas.support.events.AbstractCasEvent;
+import org.apereo.cas.ticket.TicketGrantingTicket;
+
+import lombok.Getter;
+import lombok.ToString;
+import org.apereo.inspektr.common.web.ClientInfo;
+
+/**
+ * Concrete subclass of {@code AbstractCasEvent} representing a request for SLO.
```

Comment:
```suggestion
 * Concrete subclass of {@link AbstractCasEvent} representing a request for SLO.
```

Reply:
It's a copy/paste from another class Javadoc. I will fix it in a few places.